### ISSUE TITLE
[4.0][a11y]Insufficient contrast in the badges in the modules on the Control Panel page

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#20994]
Contrast ratio = 3.32:1 (#ffffff/#868e96;). 
Minimum contrast required 4.5:1 (AA level). I suggest: #ffffff/#6e7881
See: [Contrast minimum](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

**Comment by brianteeman** [comment]
For things like this it is best to wait a week or so for the latest version of the template to be merged in OR to do the test against https://github.com/joomla/40-backend-template/

**Comment by brianteeman** [comment]
I am going to close this at this time as it is different in the latest template
